deals/module/deals.go

```diff
@@ -173,11 +173,27 @@ func (m *Module) Store(ctx context.Context, waddr string, dataCid cid.Cid, piece
 	return res, nil
 }
 
+// CalculatePieceSize calculates the data and piece size of a Cid accesible
+// by the underlying Lotus node.
+func (m *Module) CalculatePieceSize(ctx context.Context, c cid.Cid) (api.DataSize, error) {
```

Comment:
We're replacing our own logic of calculating piece sizes, to an available API from Lotus.
Originally in March we had to do it ourselves since this didn't exist.

Very soon there will be a new API for also calculating CommP, which will replace this one since will allow us to make piece size calculation and CommP independent of the replication factor.. but that's will be something when the next Lotus release appears.